Precise nodejs version, how to switch and node-gyp dependency on Windows (#1055)
PrestaShop/docs: src/content/1.7/development/compile-assets.md

## Requirements

> PrestaShop development is following tools version, such as webpack which could be incompatible with some NodeJS versions. This means that at the time we developed one version of PrestaShop, we may used an older version of NodeJS. We strongly recommend you to install a package such as ['n'](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm) to switch between versions.

Reviewer: > PrestaShop development is following tools version, such as webpack which could be incompatible with some NodeJS versions. This means that at the time we developed one version of PrestaShop, we may used an older version of NodeJS.

I think this part could benefit from som rewording for clarity

> PrestaShop development is following tools version, such as webpack which could be incompatible with some NodeJS versions. This means that at the time we developed one version of PrestaShop, we may used an older version of NodeJS. We strongly recommend you ton install a package such as ['n'](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm) to switch between versions.

Reviewer 2: ```suggestion
PrestaShop development is following tools version, such as webpack which could be incompatible with some NodeJS versions. This means that at the time we developed one version of PrestaShop, we may used an older version of NodeJS. We strongly recommend you to install a package such as ['n'](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm) to switch between versions.
```

> We use [Webpack](https://webpack.js.org/) to compile assets. Currently, you only need NodeJS 12.x ([get it here](https://nodejs.org/)), NPM will take care of it all.

Reviewer: Avoid using relative time-bound words like "Currently" or "recently" on documentation. Some people might need to compile assets in previous versions (unlikely, but who knows). You need to keep instructions on how to compile previous versions.

> On Windows, you'll probably need to install `windows-build-tools` using `npm i --global windows-build-tools` in order to make [node-gyp](https://github.com/nodejs/node-gyp#on-windows) working.

Reviewer: ````suggestion
### Windows

Windows users might need to install [windows-build-tools](https://www.npmjs.com/package/@xpack-dev-tools/windows-build-tools) in order to make [node-gyp](https://github.com/nodejs/node-gyp#on-windows) work.
````
